src/main.cpp

```diff
@@ -145,13 +145,35 @@ int T_main(int argc, TCHAR* argv[])
 		return -1;
 	}
 
-	PEImage img;
-	if (!img.loadExe(argv[1]))
-		fatal(SARG ": %s", argv[1], img.getLastError());
-	if (img.countCVEntries() == 0 && !img.hasDWARF())
-		fatal(SARG ": no codeview debug entries found", argv[1]);
+	PEImage exe, dbg, *img = NULL;
+	TCHAR dbgname[MAX_PATH];
 
-	CV2PDB cv2pdb(img);
+	if (!exe.loadExe(argv[1]))
+		fatal(SARG ": %s", argv[1], exe.getLastError());
+	if (exe.countCVEntries() || exe.hasDWARF())
+		img = &exe;
+	else
+		if (exe.areDebugSymbolsStripped())

```

Comment:
Is this check necessary? I guess checking the DBG file could always be done.

Reply:
Ok, let's remove this one.